digestparser/json_output.py

```diff
@@ -0,0 +1,179 @@
+"build JSON output from digest content"
+import os
+import time
+import copy
+from collections import OrderedDict
+import requests
+from elifetools.utils import copy_attribute
+from digestparser.utils import formatter_string, msid_from_doi
+from digestparser.jats import (parse_jats_file, parse_jats_digest, parse_jats_pub_date,
+                               parse_jats_subjects, xml_to_html)
+from digestparser.build import build_digest
+from digestparser import LOGGER
+
+
+def iiif_server_info(info_url):
+    "get the image info from the IIIF server"
+    info = {}
+    if not info_url:
+        return info
+    try:
+        response = requests.get(info_url)
+        info = response.json()
+    except Exception as exception:
+        # could be any error right now
+        LOGGER.exception("Exception in iiif_server_info for GET %s. Details: %s",
```

Comment:
I now see the log - for some reason Github was showing the old diff which had no logging.

This can later be lowered to the `warn` level if it's too annoying to see exceptions being logged all the time during testing.

Reply:
Available for modifying later if needed. It is true I have completely avoided any response status code checking here, but if there is no IIIF data then the JSON will be considered invalid anyway. If we also want to beef up the IIIF handling here in future we can do that when we run into issues.